pages/api/users/[id].ts

```diff
@@ -0,0 +1,66 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from "next";
+import { updateUser } from "../../../lib/database/users";
+import { userSchema } from "../../../models/users";
+import { findUser } from "../../../lib/database/users";
+import { StatusCodes } from "http-status-codes";
+
+/**
+ * This handles a POST request to /api/users. In Next.js, the file names are what
+ * denotes the route. Read more about requests within Next here:
+ * https://nextjs.org/docs/api-routes/response-helpers
+ *
+ */
+export const userIDHandler: NextApiHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.query == undefined) {
+    return res.status(StatusCodes.INTERNAL_SERVER_ERROR).json({ error: "Internal Server Error" });
```

Comment:
let's remove the error in the json, so just send back a string with "Internal Server Error"